app/helpers/admin_helper.rb

```diff
@@ -9,4 +9,8 @@ def format_cohort(cohort)
   def admin_navigation_structure
     @admin_navigation_structure ||= NpqSeparation::NavigationStructures::AdminNavigationStructure.new
   end
+
+  def guidance_navigation_structure
+    @guidance_navigation_structure ||= NpqSeparation::NavigationStructures::GuidanceNavigationStructure.new
+  end
```

Comment:
Should we have a `GuidanceHelper` for this? Or maybe a generic `NavigationHelper` where these methods can all live?

Doesn't feel quite right that the API guidance is in the admin helper.